src/Tizen.NUI.Components/Controls/Slider.cs

```diff
@@ -968,6 +968,48 @@ public string ValueIndicatorUrl
             }
         }
 
+        /// <summary>
+        /// Flag to decide whether the thumb snap to the nearest discrete value when the user drags the thumb or taps.
+        ///
+        /// The default value is false.
+        /// </summary>
+        /// This will be public opened later after ACR done. Before ACR, need to be hidden as inhouse API.
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public bool IsDiscrete
```

Comment:
How about this? it looks simple.
https://docs.microsoft.com/ko-kr/dotnet/csharp/programming-guide/classes-and-structs/properties#auto-implemented-properties

remove `isDiscrete` and change like this =>
`public bool IsDiscrete { get; set; }`

please consider this from next time.

Reply:
Thanks for your review.
Could you let me know how I can initialize `IsDiscrete` property as `false` value?
I'll update it right away.

Reply:
Thanks for your review.
I'll update it right away.